core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -157,9 +185,19 @@ trait AttributeComponent {
     def * = (id, ownerId, namespace, name, valueString, valueNumber, valueBoolean, valueEntityRef, listIndex, listLength, transactionId) <>(WorkspaceAttributeScratchRecord.tupled, WorkspaceAttributeScratchRecord.unapply)
   }
 
-  protected object entityAttributeQuery extends AttributeQuery[Long, EntityAttributeRecord, EntityAttributeTable](new EntityAttributeTable(_), EntityAttributeRecord)
+  protected object entityAttributeQuery extends AttributeQuery[Long, EntityAttributeRecord, EntityAttributeTable](new EntityAttributeTable(_), EntityAttributeRecord) {
+    // TODO make generic
```

Comment:
TypedType is the thing slick uses to understand how to convert scala types into database speak. But there is a further extension called BaseTypedType which is more special (and required) in some way.

I figured this out by following the implicits (inSetBind is an implicit function, I found where it is and tried calling it directly). I eventually came up with the compile error "could not find implicit value for evidence parameter of type slick.ast.BaseTypedType[OWNER_ID]"